### PR TITLE
Add syntax-checks.py to the docker image

### PR DESCRIPTION
**Pull request by boekkooi-fresh**
This allows concourse users to not have there private copy of the file all over the place.

### CODE REVIEW

**Comment by sergiughf** [comment]
:shipit:

**Comment by sergiughf** [comment]
:shipit:

File: files/syntax-checks.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        config.read(path)
+        local_groups_per_stage = {"name": ini_file}
+        groups = config.sections()
+        local_groups_per_stage["groups"] = [f for f in groups if not re.match(self.exclude_groups, f)]
+        self.groups_per_stage.append(local_groups_per_stage)
+
+    def main(self):
+        for inventory in self.inventory:
+            self._read_ini(inventory)
+        #self._read_ini("/Users/ahelal/GDrive/Projects/automation/staging.ini")
```
Comment (DiSiqueira) [inline files/syntax-checks.py]: Probably not yours, but can we remove this commented line?

**Comment by DiSiqueira** [comment]
👍

File: files/syntax-checks.py (hunk 1 above)
Comment (boekkooi-fresh) [inline files/syntax-checks.py]: I just copied from https://github.com/hellofresh/hftools/blob/master/hftools-lib/hftools/skeleton-ci/tests/syntax-checks.py

**Comment by telibol** [comment]
👍

**Pull request merged**